### PR TITLE
#58 - Added PHP Composer repository support

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #58 
Added PHP Composer repository support to Artipie core

### CODE REVIEW

**Comment by 0crat** [comment]
@olegmoz[/z](https://www.0crat.com/u/olegmoz) this pull request is too small, just 9 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @g4s8[/z](https://www.0crat.com/u/g4s8) please review this and merge or reject

File: src/main/java/com/artipie/Pie.java [hunk 1]
```diff
@@ -62,7 +63,8 @@
     private static final Map<String, Function<Storage, Slice>> ADAPTERS = new MapOf<>(
         new MapEntry<>("file", FilesSlice::new),
         new MapEntry<>("npm", sto -> new NpmSlice(new Npm(sto), sto)),
-        new MapEntry<>("gem", GemSlice::new)
+        new MapEntry<>("gem", GemSlice::new),
+        new MapEntry<>("php", storage -> new PhpComposer("", storage))
```
Comment (g4s8) [inline src/main/java/com/artipie/Pie.java:67]: @olegmoz what is the first `""` argument here? Maybe we need to move it to yaml config?

Comment (olegmoz) [reply src/main/java/com/artipie/Pie.java]: @g4s8 that's base path, figured that maybe all repo types will need to support it later. Anyway, added reading base path from config, required to change the way Slices are created. Please review

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#60` is not assigned, can't get performer

**Pull request merged**